templates/aws-vpc.template.yaml

```diff
@@ -196,98 +224,112 @@ Parameters:
     AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-8]))$
     ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-28
     Default: 10.0.216.0/21
-    Description: CIDR block for private subnet 4B with dedicated network ACL located
+    Description:
+      CIDR block for private subnet 4B with dedicated network ACL located
       in Availability Zone 4
     Type: String
   PrivateSubnetATag1:
     AllowedPattern: ^([a-zA-Z0-9+\-._:/@]+=[a-zA-Z0-9+\-.,_:/@ *\\"'\[\]\{\}]*)?$
-    ConstraintDescription: tags must be in format "Key=Value" keys can only contain
+    ConstraintDescription:
+      tags must be in format "Key=Value" keys can only contain
       [a-zA-Z0-9+\-._:/@], values can contain [a-zA-Z0-9+\-._:/@ *\\"'\[\]\{\}]
     Default: Network=Private
     Description: tag to add to private subnets A, in format Key=Value (Optional)
     Type: String
   PrivateSubnetATag2:
     AllowedPattern: ^([a-zA-Z0-9+\-._:/@]+=[a-zA-Z0-9+\-.,_:/@ *\\"'\[\]\{\}]*)?$
-    ConstraintDescription: tags must be in format "Key=Value" keys can only contain
+    ConstraintDescription:
+      tags must be in format "Key=Value" keys can only contain
       [a-zA-Z0-9+\-._:/@], values can contain [a-zA-Z0-9+\-._:/@ *\\"'\[\]\{\}]
-    Default: ''
+    Default: ""
     Description: tag to add to private subnets A, in format Key=Value (Optional)
     Type: String
   PrivateSubnetATag3:
     AllowedPattern: ^([a-zA-Z0-9+\-._:/@]+=[a-zA-Z0-9+\-.,_:/@ *\\"'\[\]\{\}]*)?$
-    ConstraintDescription: tags must be in format "Key=Value" keys can only contain
+    ConstraintDescription:
+      tags must be in format "Key=Value" keys can only contain
       [a-zA-Z0-9+\-._:/@], values can contain [a-zA-Z0-9+\-._:/@ *\\"'\[\]\{\}]
-    Default: ''
+    Default: ""
     Description: tag to add to private subnets A, in format Key=Value (Optional)
     Type: String
   PrivateSubnetBTag1:
     AllowedPattern: ^([a-zA-Z0-9+\-._:/@]+=[a-zA-Z0-9+\-.,_:/@ *\\"'\[\]\{\}]*)?$
-    ConstraintDescription: tags must be in format "Key=Value" keys can only contain
+    ConstraintDescription:
```

Comment:
My gut says this is going to cause problems. Please address the newlines throughout the template. Lines 257-258 are an example

Reply:
I tested the build in a number of different regions, and the lines here were not an issue.